src/main/java/org/jenkinsci/pipeline_steps_doc_generator/ToAsciiDoc.java

```diff
@@ -54,6 +54,7 @@ private static String stripDiv(String tagged){
         while(m.find()){
             String rep = m.group();
             rep = rep.replaceAll("\\n\\s+", "\n");
+            rep.replace("{", "&#123;");
```

Comment:
Nice solution.  Tested this locally and looks like it renders correctly inside backticks and blocks